src/allmydata/scripts/runner.py

```diff
@@ -194,7 +194,30 @@ def run():
     # doesn't return: calls sys.exit(rc)
     task.react(_run_with_reactor)
 
+
+def _setup_coverage(reactor):
+    """
+    Arrange for coverage to be collected if the 'coverage' package is
+    installed
+    """
+    try:
+        import coverage
+    except ImportError:
+        return
+
+    cov = coverage.Coverage()
+    cov.start()
+
+    def write_coverage_data(*args, **kw):
```

Comment:
This should just be called with zero arguments, considering the `addSystemEventTrigger` call below.